### PR TITLE
Update footer

### PR DESCRIPTION
**Pull request by shivanidotcom**
This PR solves the issue #16

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -239,6 +239,9 @@ <h4 class="mb-0"><strong>First Contribution Hacktoberfest 2022 List</strong></h4
         </div>
       </div>
     </div>
+    <footer style="background-color:black ; color: white;text-align: center;padding: 5px;">
```
Comment (ShehrozIrfan) [inline index.html:242]: Hey @shivanidotcom 
Add the css class to footer and then define that class in css file. 
No need for inline styles.

File: assets/css/style.css [hunk 2]
```diff
@@ -50,6 +50,13 @@ body {
   font-size: 15px;
 }
 
+footer {
+    text-align: center;
+    padding: 3px;
+    background-color: black;
+    color: white;
+
```
Comment (ShehrozIrfan) [inline assets/css/style.css:58]: remove this extra line.

**Comment by shivanidotcom** [comment]
Yea sure I have update please can you look into it.
Thanks and regards.

File: assets/css/style.css (hunk 2 above)
Comment (ShehrozIrfan) [reply assets/css/style.css:58]: @shivanidotcom remove this extra line.

**Comment by shivanidotcom** [comment]
Please check now. Updated!

File: assets/css/style.css [hunk 3]
```diff
@@ -50,6 +50,13 @@ body {
   font-size: 15px;
 }
 
+footer {
```
[the remark this answers is not in the document]
Comment (shivanidotcom) [reply assets/css/style.css:49]: okay

File: assets/css/style.css (hunk 2 above)
Comment (ShehrozIrfan) [reply assets/css/style.css:54]: @shivanidotcom Line 54 is an extra line. Remove this.

**Comment by shivanidotcom** [comment]
Removed extra line 54

**Comment by ShehrozIrfan** [comment]
@shivanidotcom 
You can follow me and star this repo for further updates.
Thanks. Have fun 👍

**Comment by shivanidotcom** [comment]
Yea sure!

**Comment by ShehrozIrfan** [comment]
Hey there!
If your contribution is excluded for this repository, then please visit the repository [Ruby Rocks](https://github.com/ShehrozIrfan/ruby-rocks/) for contribution to hacktoberfest. It contains a plenty of beginner's friendly issues accepted for hacktoberfest.

Have fun!

**Pull request merged**